FE/src/components/RegisterForm/RegisterForm.js

```diff
@@ -1,45 +1,100 @@
-import { Controller, useForm } from 'react-hook-form';
-import { Button, FormControl, InputLabel, MenuItem, Select, TextField, Typography } from '@mui/material';
-import React from 'react';
-import { Container } from '@mui/system';
-import { RegisterSchema } from 'validation/RegisterSchema';
-import { yupResolver } from '@hookform/resolvers/yup';
+import { Controller, useForm } from "react-hook-form";
+import {
+  Button,
+  FormControl,
+  InputLabel,
+  MenuItem,
+  Select,
+  TextField,
+  Typography,
+} from "@mui/material";
+import React from "react";
+import { Container } from "@mui/system";
+import { RegisterSchema } from "Utils/validation/RegisterSchema";
+import { yupResolver } from "@hookform/resolvers/yup";
 
-const RegisterForm = props => {
-    const {
-        register,
-        control,
-        handleSubmit,
-        formState: { errors }
-    } = useForm({
-        resolver: yupResolver(RegisterSchema)
-    })
+const RegisterForm = (props) => {
+  const {
+    register,
+    control,
+    handleSubmit,
+    formState: { errors },
+  } = useForm({
+    resolver: yupResolver(RegisterSchema),
+  });
 
-    const onSubmit = data => console.log(data);
+  const onSubmit = (data) => console.log(data);
```

Comment:
remove the console.log here

Reply:
thanks @Al3xDo, this file will be handled in the register-branch.